jobs/cloud_controller_ng/templates/ccng_monit_http_healthcheck.sh.erb

```diff
@@ -30,28 +30,26 @@ PORT=<%= p("cc.public_tls.port") %>
 PROTOCOL="https"
 URL="https://${HOST}:${PORT}/healthz"
 
-source /var/vcap/packages/capi_utils/monit_utils.sh
-
-echo $(date --rfc-3339=ns) 'Waiting for Cloud Controller to initially become healthy at'
-
-wait_for_server_to_become_healthy "${URL}" "<%= p("cc.api_post_start_healthcheck_timeout_in_seconds") %>"
```

Comment:
Is the removal of this initial wait period done with the idea that the health-check logic below should allow for sufficient time for CCNG to start up?  We have done some digging and couldn't find a recorded need for this initial wait period, but were curious if you did any investigation around this.

Reply:
This initial waiting was required before when the healthcheck was started as first process in the dependency chain. As it has now been moved to the end of the chain, both cc and nginx are already started.

Reply:
Just now saw that you already answered this (and our other questions) in the individual commit descriptions 😅 .  Thanks.